llvm/cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -1302,8 +1302,10 @@ if(LLVM_ENABLE_EXPERIMENTAL_DEPSCAN)
 endif()
 
 set(LLVM_ENABLE_EXPERIMENTAL_CAS_TOKEN_CACHE OFF CACHE BOOL
-    "Cache tokens using -Xclang -fcas-cache-tokens")
+    "Cache tokens using -Xclang -fexperimental-cache-lex-raw")
 if(LLVM_ENABLE_EXPERIMENTAL_CAS_TOKEN_CACHE)
+  # FIXME: Update this to check -fexperimental-cache-lex-raw before dropping
```

Comment:
Changing this will make it so that if you're using an old toolchain (i.e., from anything that doesn't have this commit) `SUPPORTS_CAS_TOKEN_CACHE` returns false.